### PR TITLE
Applied style with TailwindCSS in blog

### PR DESCRIPTION
**Pull request by cainCosmico**
This PR implements style with TailwindCSS in all views. The changes made are:
- Creation of config file tailwind.config.js and application.scss
- Dark mode
- Responsive design 

Deploy link:
https://stormy-river-00547.herokuapp.com/

### CODE REVIEW

File: app/javascript/stylesheets/application.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            @apply px-4 py-3 bg-blue-600 rounded-md text-white outline-none focus:ring-4 shadow-lg ;
+        }
+        .btn-secondary {
+            @apply px-4 py-3 bg-blue-600 rounded-md text-white outline-none focus:ring-4 shadow-lg transform;
+        }
+    }
+}
+
+
+// <div class="flex flex-wrap place-items-center h-screen">
```
Comment (estebancordoba) [inline app/javascript/stylesheets/application.scss:28]: Is it necessary to have this commented?

File: app/javascript/stylesheets/tailwind.config.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+const colors = require('tailwindcss/colors')
+// const forms = require('tailwindcss/forms')
+
+module.exports = {
+  mode: "jit",
+  purge: [
```
Comment (estebancordoba) [inline app/javascript/stylesheets/tailwind.config.js:6]: What is the `purge` function?

File: app/javascript/stylesheets/tailwind.config.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  ],
+  theme: {
+    screens: {
+      sm: '480px',
+      md: '768px',
+      lg: '976px',
+      xl: '1440px',
+    },
+    // colors: {
+    //   gray: colors.coolGray,
```
Comment (estebancordoba) [inline app/javascript/stylesheets/tailwind.config.js:32]: If these lines aren't necessary please remove

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -7,10 +7,35 @@
...
 
-  <body>
+  <body class="font-sans dark:bg-gray-700">
+    <%= render "shared/header" %>  
     <%= yield %>
+    <%= render "shared/footer" %>
   </body>
+
+  <script>
+    const body = document.getElementsByTagName('body')[0];
```
Comment (estebancordoba) [inline app/views/layouts/application.html.erb:21]: is there another way to do this?

Comment (cainCosmico) [reply app/views/layouts/application.html.erb]: In the tailwind documentation to toggle between dark and light mode you have to add or remove the class “dark” in the HTML tag, so, the script to do that was created in javascript/packs/toggle_dark_mode.js and added the tag in main application.html.erb
<%= javascript_pack_tag 'toggle_dark_mode' %>

File: app/javascript/stylesheets/tailwind.config.js (hunk 3 above)
Comment (cainCosmico) [reply app/javascript/stylesheets/tailwind.config.js]: Removed comment

File: app/javascript/stylesheets/tailwind.config.js (hunk 2 above)
Comment (cainCosmico) [reply app/javascript/stylesheets/tailwind.config.js:6]: The purge function of TailwindCSS is an optimization to production mode in a css handled in a project, the purge array receives a list of the styled files, then automatically removes unused styles to improve the performance and always be included in production mode

File: app/javascript/stylesheets/application.scss (hunk 1 above)
Comment (cainCosmico) [reply app/javascript/stylesheets/application.scss]: removed commend

**Review by estebancordoba — APPROVED** [review]

**Pull request merged**